app/build.gradle

```diff
@@ -178,6 +178,7 @@ dependencies {
       'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.4.1',
       'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.4.1',
       'org.mockito:mockito-core:2.7.22',
+      'com.github.KevinGitonga:Android-MarqueeView:472f8d63fa',
```

Comment:
Shouldn't this be ``"//third_party:asia_ivity_android_marqueeview",`` instead?

Reply:
This is for gradle dependency. Doesn't resolve this when building.

Reply:
Ah oops, not sure how I made that mistake. Thanks. :)